src/contacts/contacts.py

```diff
@@ -71,11 +71,13 @@ def get_contacts(username):
                 raise PermissionError
 
             contacts_list = contacts_db.get_contacts(username)
+            app.logger.debug("Succesfully retrieved contacts.")
             return jsonify(contacts_list), 200
-        except (PermissionError, jwt.exceptions.InvalidTokenError):
+        except (PermissionError, jwt.exceptions.InvalidTokenError) as err:
+            app.logger.error("Error retrieving contacts list: %s", str(err))
```

Comment:
This logging level should not be `error`

This situation is catching an incorrect login token.  This is the fault of the user, and the application is actually operating exactly as it is supposed to when it raises this error.

Instead it should be either `warn` (if you believe this is unexpected behavior) or `info` (because while the application is actually operating as expected when refusing incorrect authentication, it would be nice to know that the contacts could not be retrieved).  ALSO, update the log message to reflect the new log level (don't use the word 'Error'):  e.g. "Failed to retrieve contacts list:  invalid permissions"